### PR TITLE
Update monitor-health.adoc

### PR DESCRIPTION
**Pull request by Feediver1**
## Description

Resolves https://redpandadata.atlassian.net/browse/DOC-651
Review deadline: **Nov 21**

## Page previews

[Preview](https://deploy-preview-872--redpanda-docs-preview.netlify.app/current/manage/monitoring/#internal-rpc-latency)

## Checks

- [ ] New feature
- [ ] Content gap
- [ X] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/partials/monitor-health.adoc [hunk 1]
```diff
@@ -137,11 +137,12 @@ rate(redpanda_kafka_request_latency_seconds_count{redpanda_request="produce"}[5m
 
 ==== Internal RPC latency
 
-To monitor Redpanda internal RPC latency, use the xref:reference:public-metrics-reference.adoc#redpanda_rpc_request_latency_seconds[`redpanda_rpc_request_latency_seconds`] histogram. For example, create a query for the 99th percentile latency:
+To monitor Redpanda internal RPC latency, use xref:reference:public-metrics-reference.adoc#redpanda_rpc_request_latency_seconds[`redpanda_rpc_request_latency_seconds`] histogram with
```
Comment (Feediver1) [inline modules/manage/partials/monitor-health.adoc]:
```suggestion
To monitor Redpanda internal RPC latency, use the  xref:reference:public-metrics-reference.adoc#redpanda_rpc_request_latency_seconds[`redpanda_rpc_request_latency_seconds`] histogram with 
```

**Comment by jason-da-redpanda** [comment]
Hi @Feediver1 

Could we change it so the example has the label as well please 

`histogram_quantile(0.99, (sum(rate(redpanda_rpc_request_latency_seconds_bucket{redpanda_server="internal"}[5m])) by (le, provider, region, instance, namespace, pod)))`

File: modules/manage/partials/monitor-health.adoc [hunk 2]
```diff
@@ -137,11 +137,12 @@ rate(redpanda_kafka_request_latency_seconds_count{redpanda_request="produce"}[5m
...
 ==== Internal RPC latency
 
-To monitor Redpanda internal RPC latency, use the xref:reference:public-metrics-reference.adoc#redpanda_rpc_request_latency_seconds[`redpanda_rpc_request_latency_seconds`] histogram. For example, create a query for the 99th percentile latency:
+To monitor Redpanda internal RPC latency, use the  xref:reference:public-metrics-reference.adoc#redpanda_rpc_request_latency_seconds[`redpanda_rpc_request_latency_seconds`] histogram with 
+the `redpanda_server="internal"` label. For example, create a query for the 99th percentile latency:
 
 [,promql]
 ----
-histogram_quantile(0.99, (sum(rate(redpanda_rpc_request_latency_seconds_bucket[5m])) by (le, provider, region, instance, namespace, pod, redpanda_server)))
+histogram_quantile(0.99, (sum(rate(redpanda_rpc_request_latency_seconds_bucket[5m])) by (le, provider, region, instance, namespace, pod)))
```
Comment (Feediver1) [inline modules/manage/partials/monitor-health.adoc]:
```suggestion
histogram_quantile(0.99, (sum(rate(redpanda_rpc_request_latency_seconds_bucket{redpanda_server="internal"}[5m])) by (le, provider, region, instance, namespace, pod)))
```

**Review by kbatuigas — APPROVED** [review]
lgtm

**Review by jason-da-redpanda — APPROVED** [review]
all good thanks

**Pull request merged**